core/app/[locale]/(default)/account/(tabs)/addresses/add/page.tsx

```diff
@@ -57,7 +59,13 @@ const FALLBACK_COUNTRY = {
   states: [],
 };
 
-export const CustomerNewAddress = async () => {
+export const metadata = {
+  title: 'Add address',
+};
```

Comment:
Should we generate the metadata for these pages instead? Or is there something I am missing around fetching the translations?
```ts
export async function generateMetadata() {
 const t = await getTranslations('Account.Addresses.Add');

  return {
    title: t('heading'),
  }
}
```

Reply:
Noticed there are a lot of metadatas missing translations, so I will tackle this in a different PR.